packages/mask/package.json

```diff
@@ -91,7 +93,7 @@
     "jsx-jsonml-devtools-renderer": "^1.4.3",
     "millify": "^4.0.0",
     "next-tick": "^1.0.0",
-    "opensea-js": "^1.1.11",
+    "opensea-js": "^1.2.7",
```

Comment:
plz revert the change?

CC @guanbinrui

Reply:
Yes, will do.